client/tailscale/tailscale.go

```diff
@@ -154,6 +154,18 @@ func Goroutines(ctx context.Context) ([]byte, error) {
 	return get200(ctx, "/localapi/v0/goroutines")
 }
 
+// Profile returns a pprof profile of the Tailscale daemon.
+func Profile(ctx context.Context, pprofType string, sec int) ([]byte, error) {
+	var secArg string
+	if sec < 0 || sec > 300 {
+		return nil, errors.New("duration out of range")
+	}
+	if sec != 0 || pprofType == "profile" {
+		secArg = fmt.Sprint(sec)
```

Comment:
Shouldn't this be:
```go
secArg = fmt.Sprintf("&seconds=%v", sec)
```
and remove the "&seconds=" from the query string below?

Reply:
You'd think, right? But the non-profile handlers treat non-empty string seconds as being a delta:

https://cs.opensource.google/go/go/+/refs/tags/go1.17.1:src/net/http/pprof/pprof.go;l=238

And then you hit:

https://cs.opensource.google/go/go/+/refs/tags/go1.17.1:src/net/http/pprof/pprof.go;l=259;drc=refs%2Ftags%2Fgo1.17.1